diff --git a/data/tracker/meson.build b/data/tracker/meson.build
index 2921ca9..5320af1 100644
--- a/data/tracker/meson.build
+++ b/data/tracker/meson.build
@@ -1,12 +1,11 @@
-# Files needed for running LocalSearch (historically known as Tracker) inside
-# the Flatpak sandbox, for systems which don't have a suitable version of
-# LocalSearch in the host OS.
+# Files needed for running Tracker inside the Flatpak sandbox, for systems
+# which don't have a suitable version of Tracker in the host OS.
 #
 # We must export the .service files from the sandbox so they work on the
-# session bus. This means the LocalSearch domain name must correspond with the
+# session bus. This means the Tracker domain name must correspond with the
 # application ID.
 
-domain_ontologies_dir = get_option('datadir') / 'localsearch3' / 'domain-ontologies'
+domain_ontologies_dir = get_option('datadir') / 'tracker3' / 'domain-ontologies'
 dbus_services_dir = get_option('datadir') / 'dbus-1' / 'services'
 
 tracker_domain_config = configuration_data()
diff --git a/data/tracker/org.gnome.Nautilus.Tracker3.Miner.Extract.service.in b/data/tracker/org.gnome.Nautilus.Tracker3.Miner.Extract.service.in
index 23ded45..eb7a87a 100644
--- a/data/tracker/org.gnome.Nautilus.Tracker3.Miner.Extract.service.in
+++ b/data/tracker/org.gnome.Nautilus.Tracker3.Miner.Extract.service.in
@@ -1,7 +1,7 @@
 [D-BUS Service]
 Name=@application_id@.Tracker3.Miner.Extract
-Exec=/app/libexec/localsearch-extractor-3 --domain-ontology @domain_rule@
+Exec=/app/libexec/tracker-extract-3 --domain-ontology @domain_rule@
 
-# Miner details needed for localsearch-control
+# Miner details needed for tracker-control
 Path=/org/freedesktop/Tracker3/Miner/Extract
 NameSuffix=Miner.Files
diff --git a/data/tracker/org.gnome.Nautilus.Tracker3.Miner.Files.service.in b/data/tracker/org.gnome.Nautilus.Tracker3.Miner.Files.service.in
index d770619..4fa7371 100644
--- a/data/tracker/org.gnome.Nautilus.Tracker3.Miner.Files.service.in
+++ b/data/tracker/org.gnome.Nautilus.Tracker3.Miner.Files.service.in
@@ -1,7 +1,7 @@
 [D-BUS Service]
 Name=@application_id@.Tracker3.Miner.Files
-Exec=/app/libexec/localsearch-3 --domain-ontology @domain_rule@ --initial-sleep 0
+Exec=/app/libexec/tracker-miner-fs-3 --domain-ontology @domain_rule@ --initial-sleep 0
 
-# Miner details needed for localsearch-control
+# Miner details needed for tracker-control
 Path=/org/freedesktop/Tracker3/Miner/Files
 NameSuffix=Miner.Files
diff --git a/src/nautilus-tag-manager.c b/src/nautilus-tag-manager.c
index 4c1da5e..b946225 100644
--- a/src/nautilus-tag-manager.c
+++ b/src/nautilus-tag-manager.c
@@ -955,7 +955,7 @@ child_watch_cb (GPid     pid,
 static void
 export_tracker2_data (NautilusTagManager *self)
 {
-    gchar *argv[] = {"tinysparql3", "export", "--2to3", "files-starred", "--keyfile", NULL};
+    gchar *argv[] = {"tracker3", "export", "--2to3", "files-starred", "--keyfile", NULL};
     gint stdout_fd;
     GPid child_pid;
     g_autoptr (GError) error = NULL;
@@ -979,7 +979,7 @@ export_tracker2_data (NautilusTagManager *self)
                                         &error);
     if (!success)
     {
-        g_warning ("Tracker 2 migration: Couldn't run `tinysparql3`: %s", error->message);
+        g_warning ("Tracker 2 migration: Couldn't run `tracker3`: %s", error->message);
         return;
     }
 
diff --git a/test/automated/displayless/meson.build b/test/automated/displayless/meson.build
index d0a70e1..9d50c50 100644
--- a/test/automated/displayless/meson.build
+++ b/test/automated/displayless/meson.build
@@ -1,4 +1,6 @@
-tracker_sandbox = find_program('localsearch')
+trackertestutils = dependency('tracker-testutils-3.0')
+
+tracker_sandbox = find_program(trackertestutils.get_variable('command'))
 
 tests = [
   ['test-directory', [
@@ -73,7 +75,7 @@ foreach t: tracker_tests
   test(
     t[0],
     tracker_sandbox,
-    args: ['test-sandbox', '--store-tmpdir', '--index-recursive-tmpdir', test_exe],
+    args: ['--store-tmpdir', '--index-recursive-tmpdir', test_exe],
     env: [
       test_env,
       'G_TEST_BUILDDIR=@0@'.format(meson.current_build_dir()),
